src/track/serato/cueinfoimporter.cpp

```diff
@@ -0,0 +1,13 @@
+#include "track/serato/cueinfoimporter.h"
+
+#include "track/serato/tags.h"
+
+namespace mixxx {
+
+double SeratoCueInfoImporter::getTimingOffsetMillis(
+        const QString& filePath,
+        const audio::SignalInfo& signalInfo) const {
+    return SeratoTags::findTimingOffsetMillis(filePath, signalInfo);
```

Comment:
How about deleting `SeratoTags::findingTimingOffsetMillis` and moving its implementation here?

Reply:
I thought about moving the function, but we'll need that function for export, too (and export won't use the CurInfoImporter class).

Reply:
Okay, then add a comment here.